### ISSUE TITLE
foreign letters causing error during compilation

### ISSUE THREAD

**Issue by aknowel** [ManimCommunity/manim#251]
Hi,
While playing with manim I wanted to write text (in polish) and noticed that using letters "ą/ę" (right alt + a/e) causes error during compilation.
Please check when using `example_scenes/basic.py` and if you change `WriteStuff` so that letter "ą" is in variable named `example_text`, compiling it throws an error `Exception: LaTeX error converting to dvi. See log output above or the log file: media/Tex/7e3c37ceb1ce05b9.log`. I have noticed that in the corresponding .tex file there is no `\usepackage[T1]{fontenc}`.

Same problem shows when trying to compile .tex with texmaker but after adding "\usepackage[T1]{fontenc}" to .tex file in texmaker it compiles succesfully. Can you tell manim to add "\usepackage[T1]{fontenc}" to every .tex file that it generates? I think it should solve the problem.

Additional problem is that letter "ł" and "Ł" are not animated properly but I think after solving problem above it may also be solved.

Thank you very much and regards

**Comment by leotrs** [comment]
Thank you very much for bringing this to our attention. Manim should indeed support unicode characters by default. 

@philippImhof, is there a reason why `example_scenes/tex_template.tex` does not include `\usepackage[T1]{fontenc}` when `manim/tex_template.tex` does? (Also, shouldn't `example_scenes/basic.py` be using `manim/tex_template.tex`? I'm a bit confused as to why this is happening.

I believe @yoshiask also has expertise in LaTeX stuff.

**Comment by PhilippImhof** [comment]
> @PhilippImhof, is there a reason why `example_scenes/tex_template.tex` does not include `\usepackage[T1]{fontenc}` when `manim/tex_template.tex` does? 

The idea is to use a custom template, e.g. one that differs from the default template. As there was no need for the `fontenc` package in that example, I did not include it.

> (Also, shouldn't `example_scenes/basic.py` be using `manim/tex_template.tex`? I'm a bit confused as to why this is 
> happening.

`basic.py` should most definetely use the default template, as no custom template is loaded anywhere in that file. 

However, I actually wonder why the `TexTemplate` class in `manim/utils/tex.py`does not include `["UTF8", ["inputenc"]]` and `["T1", ["fontenc"]]` right after line no. 90, because the class was clearly supposed to be identical to `manim/tex_template.tex`. I am sorry, this must have slipped.

**Comment by mz71** [comment]
The way to work out this (at least for me) was to

- include inputenc
- include [polish]{babel}
- for above to work, you need to *delete* \usepackage[english]{babel}. I've used 
```
tpl = TexTemplate()
tpl.CONFIG["common_packages"].remove(["babel",["english"]])
```
in order to not change the default template. It can probably be done way easier - but still, "Zażółć gęślą jaźń" rendered just fine.

**Comment by leotrs** [comment]
> However, I actually wonder why the TexTemplate class in manim/utils/tex.pydoes not include ["UTF8", ["inputenc"]] and ["T1", ["fontenc"]] right after line no. 90, because the class was clearly supposed to be identical to manim/tex_template.tex. I am sorry, this must have slipped.

Could you please make a quick PR of this? Thanks :)

**Comment by PhilippImhof** [comment]
Yes of course. I will do so as soon as I get back from my short trip.

**Comment by Aathish04** [comment]
Closing, as this should be fixed in #257, which has been merged.
